site/de/spec/glossar.md

```diff
@@ -0,0 +1,360 @@
+
+#### Trainingsdaten
+
+Viele OCR-Tools müssen an den spezifischen Bereich der zu bearbeitenden Werke angepasst werden. Diese Anpassung an den Bereich wird als Training bezeichnet. Die Daten, die zur Steuerung dieses Prozesses verwendet werden, nennt man Trainingsdaten. Es ist wichtig, dass die Teile dieser Daten, die dem Trainingsalgorithmus zugeführt werden, manuell erfasst werden.
+
+## Verarbeitungsschritte
+
+### Binarisierung
+
+Binarisierung bedeutet die Umwandlung aller Farb- oder Graustufenpixel eines Bildes in Schwarz oder Weiß.
+
+Controlled Term: `binarized` (`comments` einer mets:file), `preprocessing/optimization/binarization` (`step` in ocrd-tool.json)
+
+Siehe [Felix Niklas' interaktive Demo](http://felixniklas.com/imageprocessing/binarization)
+
+### Border removal
+
+siehe [Cropping](#cropping)
+
+### Cropping
+
+Erkennung des Satzspiegels auf einer Seite, im Gegensatz zu den Rändern. Dies ist eine Form der
+[Regionensegmentierung](#regionensegmentierung).
+
+Controlled Term: `preprocessing/optimization/cropping`.
+
+### Deskewing
+
+Ein Bild so drehen, dass alle Textzeilen horizontal sind.
+
+Controlled Term: `preprocessing/optimization/deskewing`
+
+### Despeckling
+
+Artefakte wie Flecken, Tintenkleckse, Unterstreichungen usw. aus einem Bild entfernen. Wird üblicherweise angewendet, um 
+"Salz-und-Pfeffer"-Rauschen zu entfernen, das durch [Binarisierung](#binarisierung) entstanden ist.
+
+Controlled Term: `preprocessing/optimization/despeckling`
+
+### Dewarping
+
+Ein Bild so bearbeiten, dass alle Textzeilen begradigt und alle geometrischen Verzerrungen korrigiert sind.
+
+Controlled Term: `preprocessing/optimization/dewarping`
+
+Siehe [Matt Zuckers Eintrag zu Dewarping](https://mzucker.github.io/2016/08/15/page-dewarping.html).
+
+### Dokumentenanalyse
+
+Die Dokumentenanalyse ist die Erkennung von Strukturen auf Dokumentenebene, um z.B. ein Inhaltsverzeichnis zu erstellen.
+
+### Font-Identifizierung
+
+Erkennung der im Dokument verwendete(n) Schrift(en), entweder vor oder nach einem OCR-Prozess.
+
+Controlled Term: `recognition/font-identification`
+
+### Glyph segmentation
+
+Eine [Textzeile](#textzeile) in [Glyphen](#glyphe) unterteilen.
+
+Controlled Term: `SEG-GLYPH`
+
+### Graustufen-Normalisierung
+
+> ISSUE: [https://github.com/OCR-D/spec/issues/41](https://github.com/OCR-D/spec/issues/41)
+
+Controlled Term:
+- `gray_normalized` (`comments` in der Datei)
+- `preprocessing/optimization/cropping` (`step` in ocrd-tool.json)
+
+Die Graustufen-Normalisierung ist ähnlich wie die Binarisierung, aber statt eines rein bitonalen
+Bildes kann die Ausgabe auch Graustufen enthalten, um zu verhindern, dass versehentlich
+Glyphen kombiniert werden, wenn sie sehr nahe beieinander liegen.
+
+### Line recognition
+
+Siehe [OCR](#ocr).
+
+### OCR
+
+Abbildung von Pixelbereichen auf [Glyphen](#glyphe) und [Wörter](#wort).
```

Comment:
... und Zeilen und Regionen

Reply:
Hier wäre vielleicht noch die Frage, ob man OCR im engeren Sinn und im weiteren Sinn unterscheiden will 
i.e.S. als Teilschritt von OCR i.w.S., also die Abbildung von Pixelbereichen auf Glyphen und Wörter
und i.w.S. als Prozess mit den ganzen anderen Schritten

Dann wiederum wäre Deine Ergänzung in der zweiten Definition?

Reply:
"Im engeren Sinne", d.h. wie die Engines tatsächlich funktionieren, ist OCR das Erkennen von Zeileninhalten. Die ursprüngliche Formulierung "map pixel areas to [glyphs](#glyph) and [words](#words)" stammt von @cneud IIRC. Sie ist sehr prägnant, aber vielleicht nicht übermäßig illustrativ.

Reply:
Vielleicht wenn man statt "Abbildung von..." -> "Interpretation von Pixelbereichen als..." ?

Reply:
Ich finde das überaus illustrativ! Und ja: man kann es für beide Lesarten verwenden, die enge wie die weite, einfach indem man die Strukturebene des Gegenstandsbereiches wechselt.

Es wäre schon gut, diese Ebene im Glossar anzureißen, denn meines Erachtens haben wir tatsächlich das terminologische Problem, daß im Englischen
- `OCR` heute eher den Gesamtprozeß meint
- `Text recognition` eher den Teilschritt

wogegen im Deutschen
- `OCR` beides meinen kann
- `Texterkennung` eher den Oberbegriff meint
- `Zeichenerkennung` eher den Teilschritt,

also `Texterkennung` als falscher Freund fungiert.

Wie wäre es mit:

> Interpretation von Pixelbereichen auf [Textregionen](#region), [Zeilen](#textzeile), [Wörter](#wort) und [Zeichen](#glyphe). Meint im _engeren Sinne_ die Elementaroperation des Mustererkenners (welche früher auf Zeichen, heute auf ganzen Wörtern oder Zeilen angewandt wird), im _weiteren Sinne_ alle dazu vorab nötigen [Verarbeitungsschritte](#verarbeitungsschritte), also auch die [Segmentierung](#segmentierung) in [Satzspiegel](#satzspiegel) (d.h. [Cropping](#cropping)), [Regionen](#region) (d.h. [Regionensegmentierung](#regionensegmentierung)) und [Zeilen](#textzeile) (d.h. [Zeilensegmentierung](#zeilensegmentierung)).

Reply:
> 
> > Interpretation von Pixelbereichen auf [Textregionen](#region), [Zeilen](#textzeile), [Wörter](#wort) und [Zeichen](#glyphe). Meint im _engeren Sinne_ die Elementaroperation des Mustererkenners (welche früher auf Zeichen, heute auf ganzen Wörtern oder Zeilen angewandt wird), im _weiteren Sinne_ alle dazu vorab nötigen [Verarbeitungsschritte](#verarbeitungsschritte), also auch die [Segmentierung](#segmentierung) in [Satzspiegel](#satzspiegel) (d.h. [Cropping](#cropping)), [Regionen](#region) (d.h. [Regionensegmentierung](#regionensegmentierung)) und [Zeilen](#textzeile) (d.h. [Zeilensegmentierung](#zeilensegmentierung)).

gefällt mir sehr gut, danke @bertsky !

Reply:
vielleicht noch besser `Interpretation von ... als ...` anstelle `auf`

Reply:
:+1: